README.md

```diff
@@ -3,7 +3,7 @@
 [![License](https://img.shields.io/pypi/l/napari-tiff.svg?color=green)](https://github.com/napari/napari-tiff/raw/master/LICENSE)
 [![PyPI](https://img.shields.io/pypi/v/napari-tiff.svg?color=green)](https://pypi.org/project/napari-tiff)
 [![Python Version](https://img.shields.io/pypi/pyversions/napari-tiff.svg?color=green)](https://python.org)
-[![tests](https://github.com/GenevieveBuckley/napari-tiff/workflows/tests/badge.svg)](https://github.com/GenevieveBuckley/napari-tiff/actions)
+[![tests](https://github.com/napari/napari-tiff/workflows/test-and-deploy/badge.svg)](https://github.com/napari/napari-tiff/actions)
 [![codecov](https://codecov.io/gh/GenevieveBuckley/napari-tiff/branch/master/graph/badge.svg)](https://codecov.io/gh/GenevieveBuckley/napari-tiff)
```

Comment:
```suggestion
[![codecov](https://codecov.io/gh/napari/napari-tiff/branch/master/graph/badge.svg)](https://codecov.io/gh/napari/napari-tiff)
```
While we're here, lets fix this one too.

And also there is one more spot:
line 53, the link to open an issue:
https://github.com/napari/napari-tiff/blob/8c704181275bc21e2ba220e4bd1bf44a97b204c4/README.md?plain=1#L53

Reply:
Oh and one more place:
https://github.com/napari/napari-tiff/blob/8c704181275bc21e2ba220e4bd1bf44a97b204c4/mkdocs.yml#L7